lib/model.js

```diff
@@ -227,7 +228,7 @@ var Events = require('./events'),
                     return Vow.reject(validateResults.reduce(function (err, validationResult, k) {
                         if (validationResult !== true) {
                             if (validationResult === false) {
```

Comment:
I'm refactoring all validation now. After this you can fulfill o reject validate promise with any value